services/queue/schemas/constants.yml

```diff
@@ -113,7 +113,7 @@ scope:
     pattern: "^[\x20-\x7e]*$"
 
 # Pattern for source URLs; this rules out insecure URLs like data: or javascript:
-source-pattern: "^(https?|ssh)://"
+source-pattern: "^(https?:\/\/|ssh:\/\/|git@)"
```

Comment:
I wonder if there are any tests that we can extend to check this as well?

Reply:
Sorry about that! I had automerge enabled. I've added a test here https://github.com/taskcluster/taskcluster/pull/6483.